### ISSUE TITLE
[BUG] Installing version 0.14.8

### ISSUE THREAD

**Issue by timothy-barry** [jmschrei/pomegranate#1057]
Hello,

I am trying to install version 0.14.8 via

```
pip install pomegranate==0.14.8
```

However, I am getting the following error:

```
      Error compiling Cython file:
      ------------------------------------------------------------
      ...
      
      cpdef disable_gpu():
          global GPU
          GPU = False
      
      cdef ndarray_wrap_cpointer(void* data, numpy.npy_intp n):
           ^
      ------------------------------------------------------------
```

I am using Cython version 0.29.35.

```
cython --version
Cython version 0.29.35
```

Any help would be appreciated.

**Comment by YuriOsokin** [comment]
See this issue #1052
There is no simple solution for production. I had to migrate to version 1.0.0. It was a headache, since it is based on PyTorch, so all datatypes are tensors with specific shapes. However, it is doable.

**Comment by tiwalayo** [comment]
@YuriOsokin Can you go into more detail about what you mean by "migrate to version 1.0.0" and how you did it?

**Comment by YuriOsokin** [comment]
@tiwalayo It was a long time ago :) You need to change all data structures to pytorch vectors. So there is no more numpy or pythonic numbers. Then it works well. I also saw here (https://github.com/jmschrei/pomegranate/issues/1052#issuecomment-2177080626) that now numpy release broke the code in the old version. It will not happen in the new version, since no numpy.

**Comment by jmschrei** [comment]
There were a few issues involved here that have now been fixed in pomegranate v0.15.0. Try that out and let me know if it works. Some of these issues included numpy v2 not being as backcompatible as hoped with Cython, Cython v3 raising some weirdness, a small bug in the definitions, and Py3.12 no longer supporting nose. Regardless, please re-open if you continue to have problems with v0.15.0.
